pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -695,7 +699,7 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 			if err != nil {
 				log.Errorf("parseSystemAdapterConfig: Network with UUID %s not found: %s\n",
 					sysAdapter.NetworkUUID, err)
-				continue
+				return
```

Comment:
I Don't understand this change.. Why is return needed here? This means.. we are not processing any other adapters..  continue means we are just moving to the next adapter.. 

Can you explain why this change is needed ? And taking a step back.. What was the issue and how this change would fix it?

Reply:
When there are system adapters attached to different networks and some of the network configuration did not reach device yet, we will only have partial system adapter configuration that can be used. It will be confusing if we apply partial configuration. It should either be the latest with full system adapters configuration or the old one with full configuration.

Reply:
H... Currently, the order of parsing is:

1) Parse Physical IO
2) Parse Network Objects
3) Parse System Adapters

Here.. The error is -- When parsing SystemAdapters - we did not find Network Objects.. So, this is an error configuration. So, are you just refusing to process this set of System Adapters??

Reply:
Passing half/partial configuration is what do should not do IMHO. We should either stick with the old configuration or move to a configuration that has all the system adapters related information. That we can easily check from UI if the device has moved to the new set of changes or has fallen back to the old configuration that it has.